engine/src/main/java/org/hibernate/search/engine/search/highlighter/dsl/impl/AbstractSearchHighlighterConfigurationOptionStepImpl.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.engine.search.highlighter.dsl.impl;
+
+import java.util.Collection;
+
+import org.hibernate.search.engine.search.highlighter.SearchHighlighterConfiguration;
+import org.hibernate.search.engine.search.highlighter.dsl.HighlighterEncoder;
+import org.hibernate.search.engine.search.highlighter.dsl.HighlighterTagSchema;
+import org.hibernate.search.engine.search.highlighter.dsl.SearchHighlighterConfigurationFinalStep;
+import org.hibernate.search.engine.search.highlighter.dsl.SearchHighlighterConfigurationOptionStep;
+import org.hibernate.search.engine.search.highlighter.spi.QueryHighlighterConfigurationBuilder;
+
+public class AbstractSearchHighlighterConfigurationOptionStepImpl<T extends SearchHighlighterConfigurationOptionStep<T>>
+		implements SearchHighlighterConfigurationOptionStep<T>, SearchHighlighterConfigurationFinalStep {
+
+	protected final QueryHighlighterConfigurationBuilder highlightBuilder;
+
+	public AbstractSearchHighlighterConfigurationOptionStepImpl(QueryHighlighterConfigurationBuilder highlightBuilder) {
+		this.highlightBuilder = highlightBuilder;
+	}
+
+	@Override
+	public T boundaryChars(String boundaryChars) {
+		highlightBuilder.setBoundaryChars( boundaryChars );
```

Comment:
We don't use the `set` prefix in other builders, e.g. `org.hibernate.search.engine.search.projection.spi.DistanceToFieldProjectionBuilder`.